src/pages/FirstRun.vue

```diff
@@ -60,6 +60,16 @@ import EngineSelector from '@/components/EngineSelector.vue';
 import { Settings } from '@/config/settings';
 import { VersionEntry } from '@/k8s-engine/k8s';
 
+declare module 'vue/types/vue' {
```

Comment:
Why is this needed? Isn't `t` a globally available method for doing i18n?

Reply:
I don't fully understand why it is needed. This block was previously present in `src/pages/Integrations.vue`. When I removed that file, builds started failing for me, complaining about `this.t`. I tried adding this block to `FirstRun.vue` and it worked. It seems to be making changes to the `Vue` object, but I don't know why. Is this not common practice in JS/TS?

I see that Phil was the one who last touched these lines - I'll reach out to him.

Reply:
It sounds like these lines are here to prevent the typescript compiler from complaining when a plugin (in this case our i18n plugin) adds stuff to the `Vue` object: https://v2.vuejs.org/v2/guide/typescript.html?redirect=true#Augmenting-Types-for-Use-with-Plugins

As Phil says, this should be in `src/typings`. But I'm not sure if that's within the scope of this PR. Let me know what you think.

Reply:
The typescript compiler won't be aware that `t` has been added as a global property to Vue via our `i18n` plugin.. I think we need to add this to `src/typings` so `tsc` will be aware whenever we try to make use of `this.t()`

Reply:
Fixed in commit 63601940.